styles/components/_misc.scss

```diff
@@ -131,6 +131,46 @@
   }
 }
 
+.curSelector {
```

Comment:
@jjeffryes Missed this in the review, but can this please get its own file in the `components` folder?